gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1185,6 +1377,13 @@ properties:
     maximum: 32872
     minimum: -32872
 
+  days_to_first_event:
+    $ref:
+      - "_terms.yaml#days_to_first_event"
```

Comment:
Does this have a "common" after it?  (I have no idea if this is required or something)

Reply:
Yes I think so, updated it